### PR TITLE
A couple cross browser fixes

### LINKED ISSUE #104: IE 11: flexbox bug
image

Ha ha ha

### PR DESCRIPTION
**Pull request by stuartsan**
Fixes #104, #105 

- make the toolbar buttons be chill across browsers
- fix a firefox regression
- fix a couple potential race conditions revealed by a slow instance of IE in a VM

### CODE REVIEW

File: src/assets/styles/header.scss [hunk 1]
```diff
@@ -293,21 +293,22 @@
...
   }
 
   .toolbar-item {
     margin: 0 ($offset / 2);
-    display: flex;
-    flex-direction: column;
 
-    > * {
-      flex: 1;
+    .btn,
```
Comment (stuartsan) [inline src/assets/styles/header.scss:301]: Just took out the attempts at flexboxing the height of these items, ie was barfing on it. This is not a great solution, it is brittle, but it works.

File: src/assets/styles/layout.scss [hunk 2]
```diff
@@ -31,6 +31,7 @@
     display: flex;
     flex-direction: row;
     align-items: stretch;
+    min-height: 0;
```
Comment (stuartsan) [inline src/assets/styles/layout.scss:34]: FF regression for the event list, fixed

File: src/components/EventMap.js [hunk 3]
```diff
@@ -84,6 +84,8 @@ export default function(store){
     },
     methods: {
       plotEvents() {
```
Comment (stuartsan) [inline src/components/EventMap.js:86]: A couple potential race conditions guarded against -- if these are called but the map hasn't yet initialized, return.

**Comment by cbcowans** [comment]
👍

**Pull request merged**